StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1023DereferenceAndAccessOfSymbolsMustBeSpacedCorrectly.cs

```diff
@@ -125,40 +137,27 @@ private static void HandleAsteriskToken(SyntaxTreeAnalysisContext context, Synta
                 allowAtLineStart = false;
                 allowAtLineEnd = true;
                 allowPrecedingSpace = false;
-                var nextToken = token.GetNextToken();
-                switch (nextToken.Kind())
+                allowTrailingSpace = token.GetNextToken().Kind() switch
                 {
-                case SyntaxKind.OpenBracketToken:
-                case SyntaxKind.OpenParenToken:
-                case SyntaxKind.CloseParenToken:
-                case SyntaxKind.AsteriskToken:
-                    allowTrailingSpace = false;
-                    break;
-
-                default:
-                    allowTrailingSpace = true;
-                    break;
-                }
-
+                    SyntaxKind.OpenBracketToken or
```

Comment:
❕ binary operators go at the beginning of the next line when the expression is split